timestamp.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(express.static("./static"));
+
+app.get("/timestamp/:timeStamp", (req, res) => {
+    const TimeStamp = req.params.timeStamp;
+
+    console.log("timeStamp : "+TimeStamp);
+    let timeStamp = "-";
+
+    if(TimeStamp === "") {
+        timeStamp = (new Date()).getTime();
+        utcString = (new Date()).toUTCString();
+    }
+    else if ( !isNaN(TimeStamp) ) {
+        timeStamp = (new Date(TimeStamp)).getTime();
+        utcString = (new Date(TimeStamp)).toUTCString();
+    }
+    else {
+        res.send(JSON.stringify({"error": "Invalid Date" }));
```

Comment:
Interestingly Date function was able to decide what was invalid. Can you dig more on that?
![image](https://user-images.githubusercontent.com/5979667/56008321-833dfa80-5cf9-11e9-814e-da3d477ffe83.png)